llvm/test/CodeGen/RISCV/cheri/float-mem.ll

```diff
@@ -0,0 +1,114 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
Use a hard-float ABI for the `+f` tests as I think that'll exercise more of the backend code, whilst also removing some of the `fmv`s here that are about the float calling convention.